src/main/java/com/nextcloud/common/NextcloudClient.kt

```diff
@@ -50,48 +50,54 @@ import javax.net.ssl.SSLSession
 import javax.net.ssl.TrustManager
 
 
-class NextcloudClient(var baseUri: Uri, val context: Context) {
-    lateinit var credentials: String
-    lateinit var userId: String
-    lateinit var request: Request
-    var followRedirects = true;
-    val client: OkHttpClient
-    
+class NextcloudClient(var baseUri: Uri,
+                      var userId: String,
```

Comment:
Please note that by default, Kotlin members are `public`, contrary to Java, where default scope is package private.

We shall close the access as much as possible. If we need to keep them accessible insidde tests, `internal` scope can be used to limit access to library only.